### ISSUE TITLE
Bunyan CLI res serializer duplicates Status line of request

### ISSUE THREAD

**Issue by call-a3** [trentm/node-bunyan#444]
When the bunyan cli tool prints out a response entry in a record, it prints out the line with HTTP status codes twice. Example:

```
HTTP/1.1 500 Internal Server Error
HTTP/1.1 500 Internal Server Error
Content-Type: application/json
Date: Thu, 29 Sep 2016 15:07:46 GMT
Connection: keep-alive
```

**Comment by aiham** [comment]
This is because at https://github.com/trentm/node-bunyan/blob/05a49c34c94c020bde19f3e3f0a5f16cfdb234f4/bin/bunyan#L963 a status header is generated based on the `statusCode` response property, but there also is a status header included in the response headers, like so:

```
"res":{"statusCode":201,"header":"HTTP/1.1 201 Created\r\n
```

This existing status header gets printed as is and you end up with duplicate lines. I'm using express so that's probably where the header line is coming from.

**Comment by aiham** [comment]
@trentme Are you willing to make similar changes for this issue as proposed in https://github.com/trentm/node-bunyan/issues/504#issuecomment-308636470 ?

**Comment by trentm** [comment]
@aiham Thanks.

I think the case is simpler here. Both bunyan 1.x and 2.x could, I think, be fixed to not emit that first "HTTP/..." status line if it looks like `res.header` is a string that already has it.

**Comment by trentm** [comment]
Published bunyan@2.0.2 (beta) and bunyan@1.8.12 (latest) with this. Sorry for the delay.

**Comment by aiham** [comment]
@trentm Thanks!
